clients/admin-ui/src/features/datastore-connections/add-connection/forms/CustomInput.tsx

```diff
@@ -93,16 +94,19 @@ const CustomInput = ({
             placeholder={props.placeholder}
             resize="none"
             size="sm"
+            value={field.value || ""}
```

Comment:
Good catch. [inputs.tsx](https://github.com/ethyca/fides/blob/73bba0e11004f14aaaa81fe2ac0d9c280db97308/clients/admin-ui/src/features/common/form/inputs.tsx#L415) has this handling for all text inputs, though I guess this file uses feature-specific styling.